src/linux/Packaging.Linux/pack.sh

```diff
@@ -51,11 +55,8 @@ fi
 if [ -z "$SYMBOLS" ]; then
     die "--symbols was not set"
 fi
-
-ARCH="`dpkg-architecture -q DEB_HOST_ARCH`"
-
-if test -z "$ARCH"; then
-    die "Could not determine host architecture!"
+if [ -z "$ARCH" ]; then
+    die "--arch was not set"
 fi
```

Comment:
```suggestion
if [ -z "$RUNTIME" ]; then
    die "--runtime was not set"
fi
```

Reply:
Further down in this file we'll want to replace all instances of `linux_$ARCH` with `$RUNTIME`, except in the Debian control file where we must keep `$ARCH`.

Just before that, we can move the below code block to compute the arch:

```shell
case $RUNTIME in
    linux-x64)
        ARCH="amd64"
        ;;
    linux-arm64)
        ARCH="arm64"
        ;;
    linux-arm)
        ARCH="armhf"
        ;;
    *)
        die "Incompatible runtime architecture given for pack.sh"
        ;;
esac
```

Reply:
Sounds good. I'll commit your suggestions and then implement those remaining changes.